src/screens/Customers/AddCustomer.tsx

```diff
@@ -4,45 +4,70 @@ import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 import { Redirect, RouteComponentProps, useHistory } from 'react-router-dom';
 import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
 import Button from '../../components/Button';
 import Checkbox from '../../components/Checkbox';
 import TextField from '../../components/TextField';
-import { SiteRecord } from '../../lib/airtable/interface';
+import { SiteRecord, TariffPlanRecord } from '../../lib/airtable/interface';
 import { createCustomer } from '../../lib/airtable/request';
 import { setCurrentCustomerIdInRedux } from '../../lib/redux/customerData';
 import { selectAllTariffPlansArray } from '../../lib/redux/siteDataSlice';
-import { EMPTY_CUSTOMER } from '../../lib/redux/customerDataSlice';
+import { EMPTY_CUSTOMER, MeterType } from '../../lib/redux/customerDataSlice';
 import { selectCurrentSiteInformation } from '../../lib/redux/siteData';
+import TariffPlanCard from '../Profile/components/TariffPlanCard';
 
 const styles = (theme: Theme) =>
   createStyles({
-    content: {
-      color: theme.palette.text.primary,
-    },
     formControl: {
       width: '100%',
     },
+    disabledFormControl: {
+      width: '100%',
+      backgroundColor: theme.palette.background.default,
+    },
+    textContainer: {
+      padding: '2px 0px',
+    },
+    selectContainer: {
+      padding: '10px 0px',
+    },
+    twoColumnContainer: {
+      flexDirection: 'row',
+      display: 'flex',
+    },
+    buttonContainer: {
+      width: '100%',
+      display: 'flex',
+      justifyContent: 'center',
+      paddingTop: '10px',
+    },
+    button: {
+      width: '70%',
+    },
   });
 
 interface AddCustomerProps extends RouteComponentProps {
-  classes: { content: string, formControl: string };
-  location: any;
+  classes: { formControl: string, disabledFormControl: string, twoColumnContainer: string, textContainer: string, selectContainer: string, buttonContainer: string, button: string, };
   currentSite: SiteRecord;
+  location: any;
 }
 
-
 function AddCustomer(props: AddCustomerProps) {
   const { classes } = props;
   const history = useHistory();
 
-  const [selectedTariffPlanId, setSelectedTariffPlanId] = useState("");
   const [customerName, setCustomerName] = useState("");
-  const [hasMeter, setHasMeter] = useState(false);
-  // TODO: @julianrkung Look into constraints on meter number input.
-  const [meterNumber, setMeterNumber] = useState("");
-  const [customerInactive, setCustomerInactive] = useState(false);
+  const [customerNumber, setCustomerNumber] = useState("");
```

Comment:
I don't believe the backend API endpoint passes the `customerNumber` field when creating a customer (code [here](https://github.com/calblueprint/meepanyar-node/blob/main/index.js#L72)). Can you make a backend branch so the `customerNumber` assigned here persists to the customer record created?